leaderboards/leaderboard_challenges.tex

```diff
@@ -0,0 +1,165 @@
+    \label{fig:kick_leaderboard}
+\end{figure}
+\subsection{Challenge Execution}
+Each attempt will consist of kicking towards each of the three targets (illustrated in \cref{fig:kick_leaderboard}):
+\begin{itemize}
+    \item The centre mark in the centre circle
+    \item The penalty mark on the opposite side of the field
+    \item A corner on the opposite side of the field
+\end{itemize}
+In each attempt, for each kick, participating teams must start the robot behaviour of their competing robot via a single chest button
+press. The robot then has \qty{\PenaltyShootoutKickTime}{\second} to kick the ball towards the current target.
+The competing robot must not touch the ball a second time after the ball has clearly moved,
+otherwise the attempt ends immediately without scoring.
+Once the ball has come to a complete stop, the distance from the ball to the target is measured.
+
+Code changes are allowed when changing targets.
+\subsection{Scoring}
+After all attempts, the best score for each target will be added together and recorded in the leaderboard.
+Distances are rounded to the closest centimetre.
+
+\section{Most Passes Leaderboard}
+This leaderboard measures the highest number of successful passes a team can
+complete in a given timeframe, emphasizing passing accuracy, speed, and coordination.
+
+\subsection{Setup}
+This leaderboard challenge will take place on a standard SPL field and requires 2 competing robots,
+a Game Controller and 1 standard SPL soccer ball.
+
+Competing robots will be placed on the touchlines on opposite sides of the same half of the field,
+in line with the penalty mark. The ball will be placed on either corner of the penalty area.
+
+\begin{figure}[t]
+    \centerline{\includegraphics[width=\columnwidth]{figs/passing_leaderboard.pdf}}
+    \caption{Passing Kick Leaderboard: Robots completing the attempt are shown in blue, target area is shown in green, exclusion area is shown in magents and ball placement is shown in red}
+    \label{fig:passing_leaderboard}
+\end{figure}
+\subsection{Challenge Execution}
+In each attempt, a playing packet will be sent from the game controller to indicate the start of the attempt,
+and will begin counting down the timer.
+Teams will then have at most \qty{\MaxTimePassingLeaderboard}{\sec} to move the ball from the starting
+position to the target area while completing as many passes as possible. The centre
+1.5 metres of the field (Centre Cricle Diameter) is an exclusion zone, no robots are allowed within this area,
+if a robot enters this area, or the ball comes to a complete stop
+within this area, the run is ended immediately. If the ball is kicked off the field,
+it is placed back on the touchline where it crossed.
+
+The ref will announce each time the ball crosses the exlusion zone, and whether this will
+count as a successful pass. This is then captured by the Game Controller.
+
+Once a robot has posession of the ball within the target area, the ref will announce that the
```

Comment:
Have we ever defined what "having possession of the ball" means? Being within X centimeters of it? In that case, maybe it should be explicitly defined in the rules. I'd also propose "the robot makes any contact with the ball while the ball is in the target area" as a more easily observable condition that should maintain the spirit of "ball possession".